config/_ytt_lib/github.com/cloudfoundry/metric-proxy/config/values.yml

```diff
@@ -7,6 +7,7 @@ images:
   metric_proxy: "oratos/metric-proxy@sha256:0e933b4f7208d3d6453ee28044126392a92055a565288434f2c522bf18a44fe1"
```

Comment:
> Fixing container metrics
Previously metric_proxy failed to get container metrics when using cf7 push --strategy rolling. Now metric_proxy uses the guid instead of the app_guid to get this correctly.

As a sanity check here, did we already happen to pull in the version of metric-proxy with this fix? I'm not seeing a diff on the image reference.

Reply:
That could be the case. We assumed it hadn't made it in yet, can't remember if we made a PR to update it before but it was probably included when we annotated the image. We haven't switched over to cutting a release of metric proxy yet that will make it easier for us to track the image we're using.

Reply:
metrics have been appearing after cf push --strategy rolling for me for a couple weeks now